### PR TITLE
implemented LoginController and AuthenticationService

### PR DESCRIPTION
**Pull request by matviichuk92**

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/users/login.jsp [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+<form method="post" action="${pageContext.request.contextPath}/login">
+    <label for="login">Login:</label><br>
+    <input type="text" id="login" name="login"><br>
+    <label for="pwd">Password:</label><br>
+    <input type="password" id="pwd" name="pwd"><br>
+    <input type="submit" value="Login">
+</form>
+<br>
+<form action="${pageContext.request.contextPath}/registration">
+    <input type="submit" value="Registration!"><br></form>
```
Comment (Victoria-Moiseienko) [inline src/main/webapp/WEB-INF/views/users/login.jsp:18]: you had better use "Login!" instead of "Registration!"

**Review by Victoria-Moiseienko — COMMENTED** [review]
Good job :)

File: src/main/webapp/WEB-INF/views/users/login.jsp (hunk 1 above)
Comment (matviichuk92) [reply src/main/webapp/WEB-INF/views/users/login.jsp:18]: I am not clear what you mean.
There is exist button "Login!". "Registrations" is another button.

File: src/main/java/online/shop/security/AuthenticationServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    private UserService userService;
+
+    @Override
+    public User login(String login, String password) throws AuthenticationException {
+        User userFromDB = userService.findByLogin(login).orElseThrow(() ->
+                new AuthenticationException("Incorrect login or password!"));
+        if (userFromDB.getPassword().equals(password) && userFromDB.getLogin().equals(login)) {
+            return userFromDB;
+        }
+        throw new AuthenticationException("Incorrect login o password!");
```
Comment (dmytro-hryhoruk) [inline src/main/java/online/shop/security/AuthenticationServiceImpl.java]: either make both checks in one if() or make exception message a constant

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**